test/uk/gov/hmrc/payetaxcalculatorfrontend/services/DetailedCalcEngineSpec.scala

```diff
@@ -106,6 +106,36 @@ class DetailedCalcEngineSpec extends FreeSpec with Matchers with PropertyChecks
         }
       }
     }
-  }
```

Comment:
Yeah. Fine with this for now, and with the approach to map to the tickets. Merge and move on imho. Any violent objections ? (otherwise, I'll merge this tonight).